microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -12,34 +12,35 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, I
 
 # Understand WebView2 SDK versions  
 
-WebView2 depends on Microsoft Edge to function.  Each WebView2 SDK requires that a minimum browser version is installed.  The minimum version is reflected in the package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install Microsoft Edge with a build number of 488 or later.  The browser version is also specified in the WebView2 [Release Notes][Releasenotes].  For more information about the latest release of the Microsoft Edge browser, navigate to [Browser Channels][DeployedgeChannels].  
+To develop a WebView2 application, you must install either the [WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/) or a [non-stable Microsoft Edge channel](https://www.microsoftedgeinsider.com/en-us/download).  The minimum version is reflected in the NuGet package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install either the [WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/) or a [non-stable Microsoft Edge channel](https://www.microsoftedgeinsider.com/en-us/download) with a build number of 488 or later.  The minimum version required is also specified in the WebView2 [Release Notes][Releasenotes]. New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge (Chromium) browser, which is approximately every 6 weeks.  
 
-> [!NOTE]
-> WebView2 is currently in preview.  While the WebView team strives to ensure backward compatibility between browser versions and SDKs, it is not guaranteed because newer versions of the browser may not support previous SDK versions.  If there are breaking changes between browser versions and SDKs, the changes are specified in the [Release Notes][Releasenotes].  
-
-In the future, the WebView team plans to change the distribution model for WebView2 apps.  For more information, navigate to [Evergreen distribution mode][DistributionEvergreenMode].  
+> [!IMPORTANT]
+> When developing WebView2 applications, regularly test your application against the latest versions of the WebView2 SDK. Because the web platform is constantly evolving, regular testing is the best way to ensure your application performs as intended.
 
 ## Release and prerelease package  
 
-In preview, the release package contains the following.  
+The WebView2 Nuget package contains both a release and pre-release package.
+
+The release package is forward compatible and contains the following.  
 
-*   [Win32 C/C++ APIs][ReferenceWin32]: APIs in the SDK that are expected to remain the same at GA.  
+*   [Win32 C/C++ APIs][ReferenceWin32]: APIs in the SDK that are fully supported.  
 
-In preview, the prerelease package contains the following components.  
+The prerelease package contains the following components.  
```

Comment:
pre-release also has the win32 APIs right?

Reply:
I considered those a part of experimental APIs

Reply:
Nah we have Stable + Experimental in Pre-release.